solr-search/src/main/java/edu/unc/lib/dl/search/solr/model/BriefObjectMetadata.java

```diff
@@ -26,6 +26,10 @@
 public interface BriefObjectMetadata {
 	public CutoffFacet getAncestorPathFacet();
 
+	public ObjectPath getObjectPath();
+
+	public void setObjectPath(ObjectPath pathObject);
```

Comment:
Got a little dyslexic when I entered the name originally and looks like I missed a few places when cleaning up.  Thanks for catching, I'll fix it!